build/sources.props

```diff
@@ -3,6 +3,7 @@
     <RestoreSources>
       $(RestoreSources);
       https://api.nuget.org/v3/index.json;
+      https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
```

Comment:
Microsoft.Extensions.Logging.Testing isn't published on nuget.org

Reply:
It's intended to be for internal testing purposes only. I see that the only thing you are using in it is TestLogger in which case, I think it's best to copy that code over instead.

Reply:
I would still prefer copying code instead of adding a reference to the Logging.Testing package from our myget feed.

Reply:
I'm going to leave this using Logging.Testing for now. If the reference ever becomes an issue then we can replicate TestLogger locally.